ui/admin/tests/e2e/tests/target.spec.js

```diff
@@ -18,8 +20,11 @@ const {
   createNewHostSet,
   createNewHostInHostSet,
   createNewTarget,
+  createNewTargetWithAddress,
+  waitForSessionToBeVisible,
```

Comment:
I'm cool with refactoring the first test to make use of `connectToTarget` and `waitForSessionToBeVisible`